### PR TITLE
allow login:password@ip.ip.ip.ip:port for kodi notification

### PR DESCRIPTION
**Pull request by selios**
to allow kodi notification with http api and login password, we need to change the split ":"

### CODE REVIEW

File: amc.groovy [hunk 1]
```diff
@@ -24,7 +24,7 @@ def clean     = tryQuietly{ clean.toBoolean() }
 def exec      = tryQuietly{ exec.toString() }
 
 // array of kodi/plex/emby hosts
-def kodi = tryQuietly{ any{kodi}{xbmc}.split(/[ ,|]+/)*.split(/:/).collect{ it.length >= 2 ? [host: it[0], port: it[1] as int] : [host: it[0], port: 8080] } }
+def kodi = tryQuietly{ any{kodi}{xbmc}.split(/[ ,|]+/)*.split(/:[0-9]+/).collect{ it.length >= 2 ? [host: it[0], port: it[1] as int] : [host: it[0], port: 8080] } }
```
Comment (rednoah) [inline amc.groovy]: If the password is a number, it'll split by that. Also, your pattern will split and remove the number part. Please make sure that your split regex only matches `:` and use regex look-ahead to make sure that the port number and `$` follow.

**Comment by rednoah** [comment]
Good idea. Bad code. Please fix.

**Comment by selios** [comment]
Yes fist commit was really bad . I hope this second will be better :-)

**Comment by rednoah** [comment]
Done. Please test the latest revision.

**Comment by selios** [comment]
it's ok for me. thx

**Pull request closed without merging**